styleguide/CHANGELOG.md

```diff
@@ -12,10 +12,12 @@ NOTE: Refer to upcoming changes in our README.md under "Roadmap"
 * DTGB-650: multiple cta-block wrapper
 * DTGB-651: Links on topic overview page
 * DTGB-654: File download molecule
+* DTGB-656: Frequently visited molecule
 
 ### Updated
 
 * DTGB-651: Topic teaser styling
+* DTGB-656: Border-width of tag 
```

Comment:
According to github, you have trailing spaces here... but I don't see CodeClimate complain about it though...